docs/run-evals.md

```diff
@@ -4,12 +4,15 @@ We provide two command line interfaces (CLIs): `oaieval` for running a single ev
 
 ## Running an eval
 
-When using the `oaieval` command, you will need to provide both the model you wish to evaluate as well as the eval to run. E.g.,
+When using the `oaieval` command, you will need to provide the completion function you wish to evaluate as well as the eval to run. E.g.,
 ```sh
 oaieval gpt-3.5-turbo test-match
 ```
+The valid eval names are specified in the YAML files under `evals/registry/evals` and their corresponding implementations can be found in `evals/elsuite`.
 
-In this example, `gpt-3.5-turbo` is the model to evaluate, and `test-match` is the eval to run. The valid model names are those which you have access to via the API. The valid eval names are specified in the YAML files under `evals/registry/evals`, and their corresponding implementations can be found in `evals/elsuite`.
+`gpt-3.5-turbo` is an implementation of the [`CompletionFn` protocol](completion-fn-protocol.md). Any implementation of the `CompletionFn` protocol can be run against `oaieval`. By default, we support calling `oaieval` with any model availableon the OpenAI API or with CompletionFunctions available in [`evals/registry/completion_fns`](../evals/registry/completion_fns/). 
```

Comment:
This is a bit confusing since `gpt-3.5-turbo` is an API model, but also refers to instance of an `OpenAIChatCompletionFn`. Maybe instead say something along the lines of "gpt-3.5-turbo is an OpenAI model which we dynamically instantiate as a OpenAIChatCompletionFn(model=gpt-3.5-turbo)"